css/style.css

```diff
@@ -0,0 +1,303 @@
+html,
+.body {
```

Comment:
Tutaj nie powinno być kropki - to znacznik (tag), a nie klasa.
Przez to mamy takie brzydkie odstępy:
![image](https://github.com/devmentor-pl/task-html-and-css-basics/assets/55095257/9e469ecd-b90c-4763-a6d7-10f99e39235e)